From: Simon Glass <sjg@chromium.org>
Subject: tools: rkcommon: Add RV1106 support
Date: Tue, 07 Jul 2026 09:31:07 -0600

Add rv1106 to the list of supported SoCs. Like the RV1103B, it uses
the version-2 image header, for which the 4-byte header tag is unused.

The RV1103 is a package variant of the RV1106 and uses the same boot
image format, so this entry covers both

Signed-off-by: Simon Glass <sjg@chromium.org>
---

(no changes since v1)

 tools/rkcommon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/rkcommon.c b/tools/rkcommon.c
index 6f25a1cda31..06df57ec72d 100644
--- a/tools/rkcommon.c
+++ b/tools/rkcommon.c
@@ -154,6 +154,7 @@ static struct spl_info spl_infos[] = {
 	{ "rk3576", "RK35", 0x80000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rk3588", "RK35", 0x100000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rv1103b", "110E", 0x40000 - 0x1000, false, RK_HEADER_V2 },
+	{ "rv1106", "110C", 0x40000 - 0x1000, false, RK_HEADER_V2 },
 	{ "rv1108", "RK11", 0x1800, false, RK_HEADER_V1 },
 	{ "rv1126", "110B", 0x10000 - 0x1000, false, RK_HEADER_V1 },
 };
-- 
2.43.0

